src/utp_stream.cpp

```diff
@@ -1172,6 +1171,31 @@ void utp_stream::do_connect(tcp::endpoint const& ep)
 	m_impl->send_syn();
 }
 
+#ifdef TORRENT_UTP_LOG
```

Comment:
is it possible to put this at the top of the file, together with the other logging related functions?
also, is there a good reason these functions have to be static members? They don't need any access to private members of utp_stream. I think it would be more appropriate for them to be free functions.